pychromecast/socket_client.py

```diff
@@ -895,17 +896,18 @@ def send_message(
                     else:
                         callback_function(True, None)
                 self.socket.sendall(be_size + msg.SerializeToString())
-            except socket.error:
+            except socket.error as exc:
                 if callback_function:
                     callback_function(False, None)
                 if not no_add_request_id:
                     self._request_callbacks.pop(request_id, None)
                 self._force_recon = True
-                self.logger.info(
-                    "[%s(%s):%s] Error writing to socket.",
+                self.logger.warning(
```

Comment:
Why do we increase the severity here?

Reply:
This is a real error. It should not log secondarily due to force recon.

Reply:
Ps. This could also be some valid disconnection, which we likely should just info log. But if that is the case we can figure out that later.